### PR TITLE
Cleanup

### LINKED ISSUE #25: Clean up file names
* [ ] Name the test wiki pages consistently

### PR DESCRIPTION
**Pull request by U8NWXD**
Clean up wiki pages by:

* Adding some orphan pages to the sidebar
* Deleting some pages that we don't use anymore
* Consolidating pages that cover similar topics
* Delete trailing whitespace from the edited files

Note that there is no new content in this PR. I am just rearranging content and deleting obsolete pages. See the commit messages for the justifications behind removing each deleted page.

TODO:

* [x] Reorganize sidebar (see https://github.com/U8NWXD/oppia-wiki/pull/30#discussion_r717939851)

Fixes #25

### CODE REVIEW

**Comment by U8NWXD** [comment]
@DubeySandeep @seanlip PTAL. There's no new content here, so I don't think we need to do the usual first pass by someone familiar with the technical side of what's documented

File: How-to-do-QA.md [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-The below sections give guidelines on how to perform all the above tests.
-
-## Automated testing
-Any piece of code that we merge into develop is expected to have unit tests and end-to-end tests. Unit testing is important to make sure that each individual section of code is behaviourally correct. Make sure that the unit tests test the behaviour and not the implementation as such. Tests are expected to be written for both Frontend and Backend code.
-
-End-to-end tests, on the other hand, ensure that a combined feature functions correctly. Once a feature is complete, before it is released to the public, we expect end-to-end tests to be written for the same.
-
-***
-
-## Manual testing
```
Comment (seanlip) [inline How-to-do-QA.md:14]: Some of the things here (like the general guidelines) seem useful; could you please link to the specific part of the current wiki where it is preserved?

**Review by seanlip — CHANGES_REQUESTED** [review]
Looks good, just two notes!

File: How-to-do-QA.md (hunk 1 above)
Comment (U8NWXD) [reply How-to-do-QA.md:14]: > # Introduction
> QA is an important part of software development. Testing is typically done in two parts:
> 1. Automated tests
> 2. Manual tests
> The below sections give guidelines on how to perform all the above tests.

This is just introductory content that's already clear from the sidebar.

> ## Automated testing
> Any piece of code that we merge into develop is expected to have unit tests and end-to-end tests. Unit testing is important to make sure that each individual section of code is behaviourally correct. Make sure that the unit tests test the behaviour and not the implementation as such. Tests are expected to be written for both Frontend and Backend code.

We specify that we require full test coverage of new code in the [instructions for making a new PR](https://github.com/oppia/oppia/wiki/Make-a-pull-request#step-3-push-changes-to-your-github-fork). The requirement that we test behavior and not the implementation is documented in the wiki pages for the [frontend](https://github.com/oppia/oppia/wiki/Frontend-unit-tests-guide#test-the-interface-not-the-implementation) and [backend](https://github.com/oppia/oppia/wiki/Backend-tests#guidelines-for-writing-good-tests) tests.

> End-to-end tests, on the other hand, ensure that a combined feature functions correctly. Once a feature is complete, before it is released to the public, we expect end-to-end tests to be written for the same.

We describe E2E tests in the [E2E wiki page](https://github.com/oppia/oppia/wiki/End-to-End-Tests), and I just added a [note](https://github.com/oppia/oppia/wiki/End-to-End-Tests#write-e2e-tests) that we require E2E tests for new features.

> ***

> ## Manual testing

> Manual testing is done at 2 places in the development workflow. Every PR must be manually tested by the author and the reviewer. We expect you to test all kinds of user paths involved in the PR (and not just "happy paths", i.e. the desired user path).

This is in the [release testing quickstart guide](https://github.com/oppia/oppia/wiki/Release-testing-quickstart-guide).

> Before releasing code to users (we do monthly releases), we have a testing phase, where we have a group of people manually testing various critical user flows on Oppia. If you are interested in participating in release testing, contact @nithusha21.

This is also in the [release testing quickstart guide](https://github.com/oppia/oppia/wiki/Release-testing-quickstart-guide).

> The general guidelines for the above testing are:
> * Keep the console open at all times. There should be no console errors.
> * Try to deviate from the desired path, and make sure that any such deviation is handled appropriately.
> * Test across various browsers and platforms (if possible). At least try to cover Chrome + Firefox.
> * Test on mobile devices (actual mobile device, and not just small screen sizes on chrome). Many non-trivial bugs have been caught while testing on a mobile device.
> * For looking at console logs on mobile, you can use chrome developer tools, with USB debugging.
> * If your feature impacts learners, make sure to test with slow internet connections (Most browsers come with tools to simulate this).

These guidelines are preserved in the [release testing quickstart guide](https://github.com/oppia/oppia/wiki/Release-testing-quickstart-guide#tips-to-keep-in-mind-while-testing) (they were missing the line about mobile device testing, which I've now added).

**Review by seanlip — APPROVED** [review]
Thanks @U8NWXD !

**Review by DubeySandeep — APPROVED** [review]

**Pull request merged**